spec/reek/smells/too_many_instance_variables_spec.rb

```diff
@@ -1,32 +1,47 @@
 require_relative '../../spec_helper'
 require_lib 'reek/smells/too_many_instance_variables'
-require_relative 'smell_detector_shared'
 
 RSpec.describe Reek::Smells::TooManyInstanceVariables do
-  let(:detector) { build(:smell_detector, smell_type: :TooManyInstanceVariables) }
-
-  it_should_behave_like 'SmellDetector'
-
-  def default_max_ivars
-    Reek::Smells::TooManyInstanceVariables::DEFAULT_MAX_IVARS
-  end
+  context 'reporting smell' do
+    it 'reports the smell parameters' do
+      src = <<-EOS
+        class Empty
+          def ivars
+            @a = @b = @c = @d = 1
+            @e = 1
+          end
+        end
+      EOS
 
-  def too_many_ivars
-    default_max_ivars + 1
-  end
+      message = 'has at least 5 instance variables'
```

Comment:
I don't think we need this one-off variable - how about just putting the literal in the expectation below?